spec/integration/preview_spec.rb

```diff
@@ -579,4 +579,25 @@
       expect(err).to match_regex(/trusted option is deprecated/)
     end
   end
+
+  context 'when using --skip-inactive-nodes' do
+    env_path = File.join(testdir_simple, 'environments')
+    # deactivate node
+    test_node = "pl-inactive-node-#{rand(999999).to_i}"
+    add_node_to_puppetdb(test_node, 'production')
+    on master, puppet('node', 'deactivate', test_node)
+
+    it 'should skip inactive nodes by default' do
```

Comment:
we're explicitly using the flag here, which i also believe is the default.  is this as intended?